.travis.yml

```diff
@@ -7,6 +7,9 @@ services:
   - redis-server
 
 script:
+  - mkdir -p $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4
+  - ls -s $(pwd)/backends $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/backends
```

Comment:
@meson10 But this is working on travis, I confirmed it.
source: http://askubuntu.com/questions/56339/how-to-create-a-soft-or-symbolic-link